### PR TITLE
Authentication page(Login/SignUp)

### PR DESCRIPTION
**Pull request by malaykhakhar**

### CODE REVIEW

File: style/login.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@import url("/style.css");
+
+:root {
+  --black: black;
+}
+
+.container-auth {
+  width: 25vw;
+  /* height: 50rem; */
```
Comment (yaswanthmyneni) [inline style/login.css]: changes: remove unused code.

File: style.css [hunk 2]
```diff
@@ -20,15 +20,27 @@ html {
   justify-content: space-between;
 }
 
+.apply-flex-wrap {
```
Comment (parvez-rashid) [inline style.css:23]: Great that you've made good use of utility classes

File: style.css [hunk 3]
```diff
@@ -20,15 +20,27 @@ html {
...
 .remove-deco-white {
   text-decoration: none;
   color: var(--white);
 }
 
+.add-margin-3 {
+  margin: 3rem 0;
+}
+
 .add-gap-1 {
```
Comment (parvez-rashid) [inline style.css:36]: Great that you've made good use of utility classes

File: style.css [hunk 4]
```diff
@@ -20,15 +20,27 @@ html {
   justify-content: space-between;
 }
 
+.apply-flex-wrap {
+  flex-wrap: wrap;
+}
+
 .remove-deco-white {
```
Comment (parvez-rashid) [inline style.css:27]: Great that you've made good use of utility classes

File: pages/login.html [hunk 5]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
```
Comment (yaswanthmyneni) [inline pages/login.html:1]: UI improvement: the signup and login are not clearly visible, you can change color and increase the size a bit.
Screenshot 2022-02-21 at 3 14 03 PM
Screenshot 2022-02-21 at 3 13 53 PM

File: style/login.css [hunk 6]
```diff
@@ -0,0 +1,47 @@
+@import url("/style.css");
+
+:root {
+  --black: black;
```
Comment (parvez-rashid) [inline style/login.css]: Use hex values even for common colors for consistency.

**Review by yaswanthmyneni — APPROVED** [review]
Looks good, you can merge after changes.

File: style/login.css [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+.auth-footer a {
+  text-decoration: none;
+  color: var(--primary-color);
+}
+
+.forgot-password {
+  color: var(--primary-color);
+}
+
+.align-item {
```
Comment (parvez-rashid) [inline style/login.css:45]: Great that you've made good use of utility classes

**Review by parvez-rashid — APPROVED** [review]
UI review: The login/signup UI looks great. I've given some comments. Rest all looks good and well-structured, can be merged.

File: pages/login.html (hunk 5 above)
Comment (malaykhakhar) [reply pages/login.html:1]: Okay I will change it.

**Pull request merged**